scripts/infra/cloud-instance.sh

```diff
@@ -189,6 +189,35 @@ ec2_calculate_instance_public_dns() {
     PUBLIC_DNS=$(echo "$PUBLIC_DNS" | xargs)
 }
 
+ec2__update_ssh_config() {
```

Comment:
Any reason not to do this for all cloud types?  Seems like it could be handled generically.  Would just need to handle the dns setup, users, and keys in a general way.

Reply:
sure, it looks like supporting both isn't that bad, though I won't be able to test it with IBM cloud. i don't have it set up